.github/workflows/php-tests.yml

```diff
@@ -68,12 +76,14 @@ jobs:
     env:
       CACHEDIR: /tmp/test-cache
       WP_CORE_DIR: /tmp/wordpress
+      WP_TESTS_DIR: /tmp/wordpress-tests-lib
+      INSTALL_WP_TEST_DEBUG: true
       WP_VERSION: ${{ inputs.wordpress }}
       WP_DB_HOST: 127.0.0.1
       WP_DB_USER: root
       WP_DB_PASSWORD: '""'
       WP_MULTISITE: ${{ inputs.multisite == true && 1 || 0 }}
-
+      PACKAGE_DIRECTORY: ${{ inputs.install-core-tests == true && format('{0}/wp-content/{1}/', '/tmp/wordpress', inputs.working-directory) || '.' }
```

Comment:
I'd love a second opinion as to why using `WP_CORE_DIR` or `env.WP_CORE_DIR` does not work here. It complains the env does not exist. 🤔 

Currently, it is hardcoded to `'/tmp/wordpress'`.